blueprints/frontend.py

```diff
@@ -2,49 +2,60 @@
 
 __all__ = ()
 
-import asyncio
 import bcrypt
 import hashlib
 import os
 import time
 
 from cmyui.logging import Ansi
 from cmyui.logging import log
+from functools import wraps
+from PIL import Image
+from pathlib import Path
 from quart import Blueprint
 from quart import redirect
 from quart import render_template
 from quart import request
 from quart import session
+from quart import send_file
 
 from constants import regexes
 from objects import glob
 from objects import utils
 from objects.privileges import Privileges
-from objects.utils import flash
+from objects.utils import flash, flash_with_customizations
```

Comment:
The way I originally ordered the imports follows [this](https://www.youtube.com/watch?v=Sjor8PZXnaw&list=PLWBKAf81pmOaP9naRiNAqug6EBnkPakvY), basically:
```py
# first; stdlib imports
import stdlib_module
from stdlib_module import A
from stdlib_module import b

# second; third party imports
import third_party_module
from third_party_module import C
from third_party_module import d

# third; first party imports (imports from our project)
import first_party_module
from first_party_module import E
from first_party_module import f
```
I split up each import into it's own line to [pervent merge conflicts](https://github.com/asottile/reorder_python_imports#why-this-style), and order them alphabetically just for style.

I do this for all my projects now.